packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -416,6 +400,21 @@ export class Cluster extends Resource implements ICluster {
     }
   }
 
+  /**
+   * This method adds compute capacity to a cluster using the specified EC2 Capacity Provider.
```

Comment:
This method seems to only be for adding Fargate capacity providers -- do we want to call `addAsgCapacityProvider` from within this method to simplify the API? Otherwise the docstring should be updated.

Reply:
as per offline discussion, nixing this since `addAsgCapacityProvider` takes a capacity provider object -- only need to update docstring.